app/sfr-search-api/helpers/esSourceHelpers.js

```diff
@@ -166,6 +166,16 @@ const parseLinks = (work, nestedType) => {
               link[key] = flags[key]
             })
             delete link.flags
+
+            // Shim for compatibility with new webreader
+            // Replaces oepbs/content.opf with META-INF/container.xml and sets
```

Comment:
this only works because we know that these files are always in these locations, right

Reply:
yeah, that comment is actually a bit incorrect, it handles any `.opf` file regardless of location and we know that the `META-INF/container.xml` file is always there otherwise its an invalid epub.